### PR TITLE
CB: Filter

### PR DESCRIPTION
**Pull request by drostron**
Addresses #1593 that came out of review for #1563.

### CODE REVIEW

File: couchbase/src/test/scala/quasar/physical/couchbase/BasicQueryEnablementSpec.scala [hunk 1]
```diff
@@ -203,6 +203,24 @@ class BasicQueryEnablementSpec
       n1ql must_= """select value v from (select value object_add({}, "loc", (select value _3 from (select value loc from (select value _0 from (select value ifmissing(v.`value`, v) from `zips` v) as _1 unnest _1 as _0) as _2) as _4 unnest _4 as _3))) as v"""
     }
 
+    "convert a Eq filter" in {
+      // "select * from foo where bar 1"
```
Comment (jedesah) [inline couchbase/src/test/scala/quasar/physical/couchbase/BasicQueryEnablementSpec.scala]: I don't think the comment is inline with the test code

Comment (drostron) [inline couchbase/src/test/scala/quasar/physical/couchbase/BasicQueryEnablementSpec.scala]: True, a syntax error even.

**Comment by jedesah** [comment]
If you decide fix the comment, probably ci skip and I will use THE MERGE SCRIPT to merge in about 2 hours. Otherwise feel free to merge yourself in the meantime if this current build passes.

**Comment by drostron** [comment]
Ready

**Pull request merged**